python/hidet/ir/primitives/cpu/avx.py

```diff
@@ -43,6 +51,47 @@ def register_primitive_functions():
     for name, codegen_name, func_type in functions:
         register_primitive_function(name=name, func_or_type=func_type, codegen_name=codegen_name)
 
+    from hidet.lang import script, attrs
+    from hidet.ir.dtypes import f32x8, f32
+    from hidet.ir.func import Function
+
+    @script
+    def avx_x86_f32x8_find_sum(x: f32x8) -> f32:
```

Comment:
Is there any convention to use "find" in the function name?

If not, I would prefer to name directly as "avx_x86_f32x8_sum" and "avx_x86_f32x8_max".